src/shared/filesystemUtilities.ts

```diff
@@ -90,6 +90,10 @@ export const makeTemporaryToolkitFolder = async (...relativePathParts: string[])
 export function isInDirectory(d: string, p: string): boolean {
     const parentDirPieces = d.split(path.sep)
     const containedPathPieces = p.split(path.sep)
+    // Remove final empty element(s), if `d` ends with slash(es).
+    while (parentDirPieces.length > 0 && parentDirPieces[parentDirPieces.length - 1] === '') {
+        parentDirPieces.pop()
+    }
 
     return parentDirPieces.every((value, index) => {
```

Comment:
Isn't parentDirPieces empty by now? Would this cause false positives?

Reply:
the `parentDirPieces[parentDirPieces.length - 1] === ''` condition checks that the last element is the empty string, and only pops it in that case. Also this is covered by tests.